### ISSUE TITLE
Any way to include more discord events?

### ISSUE THREAD

**Issue by Astrobinary** [tgraupmann/ChromaDiscordApp#10]
Everything is working as intended, but I was wondering if you could possible add some more discord events specially the event to detect when your mic is muted.

I am trying to recreate a similar feature to steel series integration where I can light up a section of my keyboard if my mic is muted on discord.

https://steelseries.com/blog/steelseries-discord-gamesense-115

**Comment by tgraupmann** [comment]
These are the available RPC events that we can bind animations to.
https://discord.com/developers/docs/topics/rpc#commands-and-events-rpc-events

**Comment by Hlaxfors** [comment]
Yeah I would like to make my mute macro red when muted, how would i go about making a custom RPC event trigger?
Thanks
